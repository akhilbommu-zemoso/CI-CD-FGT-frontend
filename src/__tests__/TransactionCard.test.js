import React, { Component } from 'react'
import { render,screen } from '@testing-library/react'
import TransactionCard from '../components/TransactionsCard'

it('should take a snapshot', () => {
    const { asFragment } = render(<TransactionCard />)
    expect(asFragment(<TransactionCard />)).toMatchSnapshot()
});
it('check for Title', () => {
    const { getByText } = render(<TransactionCard />)
    expect(screen.getByText("Trasactions")).toBeInTheDocument
});

it('check for content', () => {
    const { getByText } = render(<TransactionCard />)
    expect(getByText("Looks like you didnâ€™t create any transactions to show the data. Please, here to keep in track")).toBe
})
