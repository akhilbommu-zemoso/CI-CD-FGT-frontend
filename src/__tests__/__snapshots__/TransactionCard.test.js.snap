// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiCard-root makeStyles-card-1 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      class="MuiCardContent-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
      />
      <div>
        <div
          class="makeStyles-header-3"
        >
          <p
            class="MuiTypography-root makeStyles-title-2 MuiTypography-body1"
          >
            Trasactions
          </p>
          <div>
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-iconStyle-5"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
              />
            </svg>
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-iconStyle-5"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
              />
            </svg>
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-iconStyle-5"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h6v-2H3v2zM3 6v2h18V6H3zm0 7h12v-2H3v2z"
              />
            </svg>
          </div>
        </div>
        <hr
          class="MuiDivider-root"
          style="margin-top: 10px;"
        />
      </div>
      <p />
      <p
        class="MuiTypography-root makeStyles-content-4 MuiTypography-body1"
      >
        Looks like you didnâ€™t create any transactions to show the data. Please, here to keep in track
      </p>
      <p
        class="MuiTypography-root MuiTypography-body1"
      />
    </div>
  </div>
</DocumentFragment>
`;
